aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.NSwag/TodoApi.csproj

```diff
@@ -5,6 +5,12 @@
     <ProjectUISubcaption>NSwag - ASP.NET Core 2.0</ProjectUISubcaption>
   </PropertyGroup>
 
+  <!-- <snippet_NoDocumentGeneration> -->
```

Comment:
Did you mean to import this snippet in the doc and say something about it?

Reply:
Prep for when our partner's packages support the doc gen feature and enabling / disabling it with this switch. I just happened to be testing w/ locally-build partner packages. So, no import yet.